gen/ato.py

```diff
@@ -171,10 +171,10 @@ def primary_task(self) -> Optional[FlightType]:
             FlightType.BAI,
             FlightType.DEAD,
             FlightType.SEAD,
+            FlightType.AEWC,
```

Comment:
I think same here? If a package has an E-3 and an F-15 BARCAP, the mission is BARCAP with AEW&C support.

Once we have HAVCAP that's another story, but that's a distinct mission type we need to add. I think the original placement here was correct.

Or maybe you're moving this temporarily until HAVCAP gets implemented so you can use a BARCAP _as_ a HAVCAP?

Aside from this little detail, this looks good to merge :)

Reply:
Roger. Changed it back to the intial state.